### PR TITLE
feat: add azure dev ops guide

### PR DESCRIPTION
**Pull request by raksiv**

### CODE REVIEW

File: src/pages/guides/getting-started/azure-pipelines.mdx [hunk 1]
```diff
@@ -0,0 +1,135 @@
+export const description =
+  'Deploy to AWS, Google Cloud or Microsoft Azure using Azure DevOps and the Nitric CLI'
+
+### Deployment Automation with Azure Pipelines and Nitric
+
+This guide will illustrate how Nitric can be integrated with Azure Pipelines to establish a continuous deployment pipeline. The example below focuses on AWS, but with minor adjustments, it can be adapted for Google Cloud or Microsoft Azure.
```
Comment (tjholm) [inline src/pages/guides/getting-started/azure-pipelines.mdx:6]: Guide looks good, only thought is that it's a bit strange for us to do an AWS example with Azure Devops.

Comment (raksiv) [reply src/pages/guides/getting-started/azure-pipelines.mdx:6]: Yeh fair call, will add steps for Azure as the primary.

File: src/nav.config.ts [hunk 2]
```diff
@@ -194,6 +194,10 @@ const fullNav: FullNav = {
           title: 'Using GitHub Actions',
           href: '/guides/getting-started/github-actions',
         },
+        {
+          title: 'Using Azure Devops',
```
Comment (davemooreuws) [inline src/nav.config.ts:198]: Should we title this as Azure Pipelines instead? Just a thought

File: src/pages/guides/getting-started/azure-pipelines.mdx [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+<Note>
+  This guide assumes basic knowledge about Azure Pipelines. If you're
+  unfamiliar, it's recommended to go through [Azure Pipelines
+  documentation](https://docs.microsoft.com/en-us/azure/devops/pipelines/get-started/what-is-azure-pipelines)
+  first.
+</Note>
+
+#### Pipeline Setup
+
+Firstly, ensure you have a Nitric project prepared for deployment. If not, refer to Nitric's [quickstart guide](https://nitric.io/docs/).
```
Comment (davemooreuws) [inline src/pages/guides/getting-started/azure-pipelines.mdx:17]:
```suggestion
Firstly, ensure you have a Nitric project prepared for deployment. If not, refer to Nitric's [quickstart guide](/guides/getting-started/quickstart).
```

**Review by tjholm — APPROVED** [review]

File: src/pages/guides/getting-started/azure-pipelines.mdx [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+pool:
+  vmImage: ubuntu-latest
+
+steps:
+  - task: NodeTool@0
+    inputs:
+      versionSpec: '10.x'
+    displayName: 'Install Node.js'
+
+  - script: |
```
Comment (davemooreuws) [inline src/pages/guides/getting-started/azure-pipelines.mdx:36]: Is this npm install or node required?

File: src/pages/guides/getting-started/azure-pipelines.mdx [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+```
+
+### Installing Dependencies
+
+Use the following step to install project dependencies for your project, we used `npm install` here for a node project:
+
+```yaml
+- script: |
+    npm install
+  displayName: 'npm install and build'
```
Comment (davemooreuws) [inline src/pages/guides/getting-started/azure-pipelines.mdx:109]: as above

**Review by davemooreuws — CHANGES_REQUESTED** [review]

File: src/nav.config.ts (hunk 2 above)
Comment (raksiv) [reply src/nav.config.ts:198]: Sure sounds good!

**Comment by davemooreuws** [comment]
try with this slimmed down config @raksiv :

```yml
trigger:
  - main

pool:
  vmImage: ubuntu-latest

steps:
  - script: |
      # Install Pulumi
      curl -fsSL https://get.pulumi.com | sh
      # Export path for pulumi (assuming default installation path)
      export PATH=$PATH:$HOME/.pulumi/bin
      pulumi version

      # Install Nitric
      curl https://nitric.io/install | bash
      export PATH=$PATH:$HOME/.nitric/bin

      # Login to Azure (assuming using Service Principal and Certificate)
      az login --service-principal -u $(AZURE_CLIENT_ID) -p $(AZURE_CERTIFICATE_PATH) --tenant $(AZURE_TENANT_ID)

      # Execute the nitric command
      nitric up --ci -v2
    displayName: "Install Pulumi and Nitric CLI"
    env:
      PULUMI_CONFIG_PASSPHRASE: $(PULUMI_CONFIG_PASSPHRASE)
      PULUMI_ACCESS_TOKEN: $(PULUMI_ACCESS_TOKEN)
      AZURE_CLIENT_ID: $(AZURE_CLIENT_ID)
      AZURE_TENANT_ID: $(AZURE_TENANT_ID)
      AZURE_CERTIFICATE_PATH: $(AZURE_CERTIFICATE_PATH)
```

**Review by davemooreuws — CHANGES_REQUESTED** [review]
Just the h3 to h1 change then all good

File: src/pages/guides/getting-started/azure-pipelines.mdx [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+This guide will illustrate how Nitric can be integrated with Azure Pipelines to establish a continuous deployment pipeline. The example below focuses on AWS, but with minor adjustments, it can be adapted for Google Cloud or Microsoft Azure.
+
+<Note>
+  This guide assumes basic knowledge about Azure Pipelines. If you're
+  unfamiliar, it's recommended to go through [Azure Pipelines
+  documentation](https://docs.microsoft.com/en-us/azure/devops/pipelines/get-started/what-is-azure-pipelines)
+  first.
+</Note>
+
+#### Pipeline Setup
```
Comment (davemooreuws) [inline src/pages/guides/getting-started/azure-pipelines.mdx:15]:
```suggestion
## Pipeline Setup
```

**Review by davemooreuws — CHANGES_REQUESTED** [review]

**Review by davemooreuws — APPROVED** [review]

**Review by davemooreuws — APPROVED** [review]

**Pull request merged**